modules/drivers/mongo/deps.edn

```diff
@@ -2,4 +2,6 @@
  ["src" "resources"]
 
  :deps
- {com.novemberain/monger {:mvn/version "3.6.0"}}}
+ {com.google.guava/guava {:mvn/version "32.1.3-jre"}
```

Comment:
I see `com.google.guava/guava                    {:mvn/version "32.1.2-jre"}` in `metabase/deps.edn` on `master`. Shouldn't that be updated?

Reply:
Doesn't the mongo driver inherit the root dependency? I would prefer not having to maintain the same thing at multiple places.